### PR TITLE
add dedicated LV for /var/lib/systemd/coredump

### PR DESCRIPTION
**Pull request by umezawatakeshi**
part of https://github.com/cybozu-go/neco/issues/1677

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: ignitions/common/files/opt/sbin/setup-var [hunk 1]
```diff
@@ -81,12 +87,14 @@ elif ls /dev/mapper/crypt-nvme* >/dev/null 2>&1; then
     CONTAINERD_SIZE=1t
     DOCKER_SIZE=50g
     KUBELET_SIZE=100g
+    COREDUMP_SIZE=50g
```
Comment (ysksuzuki) [inline ignitions/common/files/opt/sbin/setup-var:90]: Why is `COREDUMP_SIZE` 50G for cs and 20G for ss? I read the argument that 10G or more is enough though.

**Comment by ysksuzuki** [comment]
JVM crash dumps suppose to be stored `/var/lib/systemd/coredump` for Java applications such as Elasticsearch?

**Comment by ysksuzuki** [comment]
> JVM crash dumps suppose to be stored /var/lib/systemd/coredump for Java applications such as Elasticsearch?

Sorry , forget it. JVM crush dump is not in our scope.

File: ignitions/common/files/opt/sbin/setup-var (hunk 1 above)
Comment (umezawatakeshi) [reply ignitions/common/files/opt/sbin/setup-var:90]: the partition should be larger enough to hold uncompressed core.

**Review by ysksuzuki — APPROVED** [review]

**Pull request merged**